Blog post introducing our mentors 2023-2024 (#682)
ropensci/roweb3: content/blog/2023-11-24-r-intro-mentors/index.en.md

---
slug: "champions-program-mentors-2023"
title: Introducing rOpenSci Mentors - Cohort 2023-2024
author:

> - Yanina Bellini Saibene

Reviewer: Should mentors also be authors?

> date: '2023-11-24'

Reviewer: Don't forget to update this and the folder :grin:

tags:
- community
- champions program
description: 
---

> One important aspect of rOpenSci mission is to build capacity and promote passionate community members who help the open science and open source software community grow and improve.

Reviewer: ```suggestion
One important aspect of rOpenSci's mission is to build capacity and promote passionate community members who help the open science and open source software community grow and improve.
```

> rOpenSci champions program is a unique opportunity for us to identify, recognize, and reward such individuals and support them in their work to improve their communities and the wider ecosystem. Mentors play a vital role in the rOpenSci Champions Program by helping select Champions and advising and inspiring their mentees. They also help to shape this program by providing feedback and proposing activities and tools.   

Reviewer: ```suggestion
The rOpenSci Champions Program is a unique opportunity for us to identify, recognize, and reward such individuals and support them in their work to improve their communities and the wider ecosystem. Mentors play a vital role in the rOpenSci Champions Program by helping select Champions and advising and inspiring their mentees. They also help to shape this program by providing feedback and proposing activities and tools.   
```

> rOpenSci champions program is a unique opportunity for us to identifying, recognizing, and rewarding such individuals and suppor them in their work to improve their communities and the wider ecosystem. Mentors play a vital role in the rOpenSci Champions Program by helping select Champions and advising and inspiring their mentees. They also help to shape this program by providing feedback and proposing activities and tools.   

Reviewer 2: ```suggestion
rOpenSci champions program is a unique opportunity for us to identify, recognize, and reward such individuals and support them in their work to improve their communities and the wider ecosystem. Mentors play a vital role in the rOpenSci Champions Program by helping select Champions and advising and inspiring their mentees. They also help to shape this program by providing feedback and proposing activities and tools.   
```

> This year, rOpenSci Team reviewed 28 applications from community member in various disciplines and from different countries to select ten mentors.

Reviewer: ```suggestion
This year, the rOpenSci Team reviewed 28 applications from community member in various disciplines and from different countries to select ten mentors.
```

We are pleased to introduce you to our mentors for the cohort 2023-2024!

### Bruna Wundervald

{{< figure src = "/img/team/bruna-wundervald.jpeg" width = "300" alt = "Profile photo of Bruna Wundervald" caption = "<strong>Bruna Wundervald</strong>">}}

I am a Data Scientist graduated in Statistics, with a Ph.D. in Bayesian Machine Learning from the Hamilton Institute, Maynooth, Ireland.

My main interests are causality and experimentation, Bayesian inference, multivariate analysis, feature extraction, and natural language processing.

I am happy to be giving back to the community that has helped me immensely in the past :)

### Eunseop Kim

{{< figure src = "/img/team/eunseop-kim.jpg" width = "300" alt = "Profile photo of Eunseop Kim." caption = "<strong>Eunseop Kim</strong>">}} 

I am a data scientist at Eli Lilly and Company, specializing in the development of AI/ML methods for sales and marketing analytics. I hold a Ph.D. in Statistics from The Ohio State University, USA, and have degrees in Economics, both a Master's and a Bachelor's, from Seoul National University, South Korea. Originally from South Korea, I currently reside in the USA.

> I enjoy developing and sharing software for reproducible scientific computing. Throughout my academic and professional experience, I have been actively involved in the creation and maintenance of R packages and Shiny applications. Additionally, I am the author of the "melt" package and have served as an rOpenSci package reviewer.

Reviewer: ```suggestion
I enjoy developing and sharing software for reproducible scientific computing. Throughout my academic and professional experience, I have been actively involved in the creation and maintenance of R packages and Shiny applications. Additionally, I am the author of the "melt" package and have served as an rOpenSci software peer reviewer.
```

> As a mentor of the Champions Program, I look forward to sharing my experiences and knowledge, contributing to the ongoing success of the rOpenSci community.|

Reviewer: ```suggestion
As a mentor in the Champions Program, I look forward to sharing my experiences and knowledge, contributing to the ongoing success of the rOpenSci community.|
```

### Emi Tanaka

{{< figure src = "/img/team/emi-tanaka.jpg" width = "300" alt = "Profile photo of Emi Tanaka." caption = "<strong>Emi Tanaka</strong>">}}

Dr. Emi Tanaka is a Senior Lecturer in Statistics at the Biological Data Science Institute (and the Research School of Finance, Actuarial Studies and Statistics) at the Australian National University.

> She has developed numerous open-source tools, primarily as R-packages, and resources aimed at making statistical methods accessible to a diverse audience. Emi demonstrates a proactive approach to community development and education through her involvement in the branches of the Statistical Society of Australia (SSA) and other committees.

Reviewer 3: ```suggestion
She has developed numerous open-source tools, primarily as R packages, and resources aimed at making statistical methods accessible to a diverse audience. Emi demonstrates a proactive approach to community development and education through her involvement in the branches of the Statistical Society of Australia (SSA) and other committees.
```

Emi highly values research software as an enabler of science and as part of encouraging software development in the statistical community, she created the Di Cook Award (an open-source statistical software award) for students.

### François Michonneau

{{< figure src = "/img/team/françois-michonneau.JPG" width = "300" alt = "Profile photo of François Michonneau." caption = "<strong>François Michonneau</strong>">}}

> I am an educator specializing in modern data systems. My interest in data education started when I was working on my PhD in marine biology at the University of Florida. After leaving academia, I worked at The Carpentries, and I am currently a Senior Training Engineer at Voltron Data. I co-authored several R packages, and one of them went through the rOpenSci review process.

Reviewer: ```suggestion
I am an educator specializing in modern data systems. My interest in data education started when I was working on my PhD in marine biology at the University of Florida. After leaving academia, I worked at The Carpentries, and I am currently a Senior Training Engineer at Voltron Data. I co-authored several R packages, one of which went through the rOpenSci software peer review process.
```

> I am an educator specializing in modern data systems. My interest in data education started when I was working on my PhD in marine biology at the University of Florida. After leaving academia, I worked at The Carpentries, and he is currently a Senior Training Engineer at Voltron Data. I co-authored several R packages, and one of them went through the rOpenSci review process.

Reviewer 2: ```suggestion
I am an educator specializing in modern data systems. My interest in data education started when I was working on my PhD in marine biology at the University of Florida. After leaving academia, I worked at The Carpentries, and I am currently a Senior Training Engineer at Voltron Data. I co-authored several R packages, and one of them went through the rOpenSci review process.
```

I'm excited to be a mentor to help others participate in and grow the R community.

### Francisco Cardozo

{{< figure src = "/img/team/francisco-cardozo.JPEG" width = "300" alt = "Profile photo of Francisco Cardozo." caption = "<strong>Francisco Cardozo</strong>" >}}

> My name is Francisco Cardozo. I am currently a Ph.D. student at the University of Miami, where I am part of the Prevention Science and Community Health program.

Reviewer: ```suggestion
My name is Francisco Cardozo. I am currently a Ph.D. student at the University of Miami, USA, where I am part of the Prevention Science and Community Health program.
```

I have worked on evaluating prevention programs in various countries across Latin America. At the moment, I am developing workshops to teach the use of software that enhances the reproducibility and rigor of research. These workshops focus on the use of R and statistical analysis techniques, such as machine learning and latent variables.

I am deeply committed to strengthening technical capacities within communities; I believe in equipping people with the necessary tools to execute their projects in the best way possible.

### Luis D. Verde Arregoitia

{{< figure src = "/img/team/luis-verde.jpeg" width = "300" alt = "Profile photo of Luis D. Verde Arregoitia." caption = "<strong>Luis D. Verde Arregoitia</strong>">}}

> Postdoctoral researcher in evolutionary macroecology. Mammalogist . Biology background (BSc. Biology, Universidad Nacional Autónoma de México; PhD in Conservation Science, University of Queensland).

Reviewer 3: Would it make sense to rephrase this as sentences?

> Postdoctoral researcher in evolutionary macroecology. Mammalogist . Biology background (BSc. Biology, Universidad Nacional Autónoma de México; PhD in Conservation Science, University of Queensland).

Author: ```suggestion
I'm a postdoctoral researcher in evolutionary macroecology. I'm a mammalogist with a Biology background. I have a BSc in Biology from the Universidad Nacional Autónoma de México and a Ph.D. in Conservation Science, from the University of Queensland.
```

> I'm a postdoctoral researcher in evolutionary macroecology. I'm a mammalogist with a Biology background. I have a BSc in Biology from the Universidad Nacional Autónoma de México and a Ph.D. in Conservation Science, from the University of Queensland.

Reviewer: ```suggestion
I'm a postdoctoral researcher in evolutionary macroecology. I'm a mammalogist with a Biology background. I have a BSc in Biology from the Universidad Nacional Autónoma de México and a Ph.D. in Conservation Science, from the University of Queensland, Australia.
```

In addition to my research I spend time writing tools, blog posts, and teaching materials to help others work with messy data and spreadsheets (unheadr package) and improve their R scripts for reproducibility (annotater package).

> Mentor for the Champions program. Happy to guide others in their software development journey in a way that benefits both users and developers and ultimately enriches the open tools community in terms of packages and people.

Reviewer 4: the "Mentor for the Champions program" sentence here is redundant

> Mentor for the Champions program. Happy to guide others in their software development journey in a way that benefits both users and developers and ultimately enriches the open tools community in terms of packages and people.

Author: ```suggestion
Happy to guide others in their software development journey in a way that benefits both users and developers and ultimately enriches the open tools community in terms of packages and people.
```

### Erle Holgersen

{{< figure src = "/img/team/erle-holgersen.jpg" width = "300" alt = "Profile photo of Erle Holgersen." caption = "<strong>Erle Holgersen</strong>">}}

Hi! I'm Erle Holgersen, and hope to serve as a mentor in R package development. I am a computational biologist at Deep Genomics, where I collaborate closely with lab teams and build tools for streamlining data analysis.

> Looking forward to helping with the Champions program!

Reviewer: ```suggestion
Looking forward to helping with the Champions Program!
```

### Marie C. Vendettuoli

{{< figure src = "/img/team/marie-vendettuoli.jpg" width = "300" alt = "Profile photo of Marie C Vendettuoli" caption = "<strong>Marie C. Vendettuoli </strong>">}}

> Marie Vendettuoli (she/her) holds a PhD in Bioinformatics/Computational Biology & Human Computer Interaction from Iowa State University. She has spent over a decade developing R packages for scientists subject to regulatory oversight, both in the animal and human health domains. Marie values the challenge of bringing open source R tools to new audiences and looks forward to working with Champions to address their projects with an emphasis on curiosity. A PNW native, Marie is in the process of relocating to the Portland area (USA) after several years working in Seattle.

Reviewer: ```suggestion
Marie Vendettuoli (she/her) holds a PhD in Bioinformatics/Computational Biology & Human Computer Interaction from Iowa State University, USA. She has spent over a decade developing R packages for scientists subject to regulatory oversight, both in the animal and human health domains. Marie values the challenge of bringing open source R tools to new audiences and looks forward to working with Champions to address their projects with an emphasis on curiosity. A Pacific Northwest native, Marie is in the process of relocating to the Portland area (USA) after several years working in Seattle.
```

> Marie Vendettuoli (she/her) holds a PhD in Bioinformatics/Computational Biology & Human Computer Interaction from Iowa State University. She has spent over a decade developing R packages for scientists subject to regulatory oversight, both in the animal and human health domains. Marie values the challenge of bringing open source R tools to new audiences and looks forward to working with Champions to address their projects with an emphasis on curiosity. A PNW native, Marie is in the process of relocating to the Portland area (USA) after several years working in Seattle.

Reviewer: The term 'native' could refer to someone who spent their formative years in an area, but it could also mean someone who's indigenous (first nations or native american). Do you think it matters? Should we ask Marie if she wants to clarify?

Author: @mariev can you please let us know what do you think about Steffi comment and if you would like to do any changes?  Thanks!

### Jonathan Keane

{{< figure src = "/img/team/jonathan-keane.png" width = "300" alt = "Profile photo of Jonathan Keane." caption = "<strong>Jonathan Keane</strong>">}} 

Jonathan Keane is an engineering leader with a background in software engineering and data science. Jonathan is a maintainer of the Arrow project including the Arrow R package. Jonathan lives in Chicago, IL, USA and has a PhD in Linguistics from the University of Chicago.

### Apoorv Anand

{{< figure src = "/img/team/apoorv-anand.jpeg" width = "300" alt = "Profile photo of Apoorv Anand." caption = "<strong>Apoorv Anand</strong>">}}

> Member of CivicDataLab and Justice Hub.  Apoorv started his "open data" journey as a volunteer, with DataKind, to do "Data-for-good". He joined CivicDataLab team, in 2018, and started collaborating to make public datasets more accessible to citizens.

Reviewer: ```suggestion
Member of CivicDataLab and Justice Hub.  Apoorv started his "open data" journey as a volunteer, with DataKind, to do "Data-for-good". He joined CivicDataLab team in 2018, and started collaborating to make public datasets more accessible to citizens.
```

"I would like to contribute by not just being an active user of packages but also by helping others develop them and write about their process and experiences. This program will surely help me to grow as a developer and a mentor and will provide an opportunity to learn from and give back to this community."

> 

Reviewer: I think perhaps a final concluding sentence? Something like, "We're so happy to have this wonderful group of mentors for the 2023-2024 cohort of the Champions Program. We are currently finalizing the selection of mentees and are looking forward to announcing them next!" (if that's true, that is 😉 )

> 

Author: ```suggestion

We're so happy to have this wonderful group of mentors for the 2023-2024 cohort of the Champions Program. We are currently finalizing the selection of mentees and are looking forward to announcing them next!
```

---
slug: "champions-program-mentors-2023"
title: Introducing rOpenSci Mentors - Cohort 2023-2024
author:

> - Yanina Bellini Saibene

Author: ```suggestion
- Apoorv Anand
- Bruna Wundervald
- Eunseop Kim
- Emi Tanaka
- François Michonneau
- Francisco Cardozo
- Luis D. Verde Arregoitia
- Erle Holgersen
- Marie C Vendettuoli
- Jonathan Keane
- Yanina Bellini Saibene
```

---
slug: "champions-program-mentors-2023"
title: Introducing rOpenSci Mentors - Cohort 2023-2024
author:
- Apoorv Anand
- Bruna Wundervald
- Eunseop Kim
- Emi Tanaka
- François Michonneau
- Francisco Cardozo

> - Luis D. Verde Arregoitia

Author: ```suggestion
- Luis Verde Arregoitia
```

### François Michonneau

{{< figure src = "/img/team/françois-michonneau.jpg" class = "pull-right" width = "300" alt = "Profile photo of François Michonneau." caption = "<center><strong>François Michonneau</strong></center>">}}

I am an educator specializing in modern data systems. My interest in data education started when I was working on my PhD in marine biology at the University of Florida. After leaving academia, I worked at The Carpentries, and he is currently a Senior Training Engineer at Voltron Data. I co-authored several R packages, and one of them went through the rOpenSci review process. I'm excited to be a mentor to help others participate in and grow the R community.

### Francisco Cardozo

{{< figure src = "/img/team/francisco-cardozo.jpeg" class = "pull-right" width = "300" alt = "Profile photo of Francisco Cardozo." caption = "<center><strong>Francisco Cardozo</strong></center>">}}

> My name is Francisco Cardozo. I am currently a Ph.D. student at the University of Miami, where I am part of the Prevention Science and Community Health program. I have worked on evaluating prevention programs in various countries across Latin America. At the moment, I am developing workshops to teach the use of software that enhances the reproducibility and rigor of research. These workshops focus on the use of R and statistical analysis techniques, such as machine learning and latent variables. I am deeply committed to strengthening technical capacities within communities; I believe in equipping people with the necessary tools to execute their projects in the best way possible.

Reviewer 5: OK!
